modules/ossm-rn-fixed-issues.adoc

```diff
@@ -14,11 +14,11 @@ Provide the following info for each issue if possible:
 *Result* - How has the behavior changed as a result? Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
 ////
 
-The following issues been resolved in the current release:
-
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
```

Comment:
I'm not sure it's necessary to have a heading right after a heading here with nothing in between, but that's probably outside the scope of this PR. Personally I'd rename the whole file to **{SMProductShortName} fixed issues** instead.
![image](https://github.com/openshift/openshift-docs/assets/40172997/87896400-90dc-4f1c-b105-774d52310b0e)